modin/core/execution/ray/common/utils.py

```diff
@@ -82,7 +83,10 @@ def initialize_ray(
     # the `pandas` module has been fully imported inside of each process before
     # any execution begins:
     # https://github.com/modin-project/modin/pull/4603
-    env_vars = {"__MODIN_AUTOIMPORT_PANDAS__": "1"}
+    env_vars = {
+        "__MODIN_AUTOIMPORT_PANDAS__": "1",
+        "PYTHONWARNINGS": "ignore::FutureWarning",
```

Comment:
I'm wondering why `with filter_warnings():` inside of kernels doesn't work. Why wouldn't we just add the warning-catcher to the lowest level possible (inside the kernel that runs all other kernels), for example here:
https://github.com/modin-project/modin/blob/ea8088af4cadfb76294e458e5095f262ca85fea9/modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py#L379

Reply:
I'll try to do this for the unidist, while setting variables does not work without `mpiexec` (at least I don't know how). cc @YarShev

Reply:
done